openreview/openreview.py

```diff
@@ -419,26 +428,26 @@ def post(self, client):
         client.post_group(self)
 
 class Invitation(object):
-    def __init__(self, id, writers=None, invitees=None, noninvitees=None, readers=None, nonreaders=None, reply=None, web=None, process=None, signatures=None, duedate=None, cdate=None, rdate=None, ddate=None, multiReply=None, taskCompletionCount=None):
+    def __init__(self, id, writers=None, invitees=None, noninvitees=None, readers=None, nonreaders=None, reply=None, replyto=None, forum=None, web=None, process=None, signatures=None, duedate=None, cdate=None, rdate=None, ddate=None, multiReply=None, taskCompletionCount=None):
 
         default_reply = {
-            'forum': None,
-            'replyto': None,
+            'forum': forum,
+            'replyto': replyto,
             'readers': {'values': ['everyone']},
-            'signatures': {'values-regex': '.*'},
-            'writers': {'values-regex': '.*'},
+            'signatures': {'values-regex': '~.*'},
```

Comment:
I agree about having default values, but I'm not sure if the library is the right place, I don't want to give it more responsibility that being a bridge between the API and the users. I believe the scripts repo should have this kind of helpers to create default configurations. We can have a conversation about this.